CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.1.0
+
+New features:
+  * Add env parameter in request header.
```

Comment:
Not good at all. We want to be able to add ANY kind of custom properties (starting with underscore) like graylog documentation says it's possible. "env" is not a common use, it's just for your project and should not appear in our bundle